TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -55,6 +55,7 @@ public class TypeCobolConfiguration : ITypeCobolCheckOptions
         public string RawExecToStep = "6";
         public string RawMaximumDiagnostics;
         public string RawOutputFormat = "0";
+        public Boolean IsCobolFile;
```

Comment:
Maybe use `bool` instead of `Boolean` for a more C#-oriented style 😄.
Also this piece of information may be attached to `TextSourceInfo` instead of option. I think `IsCobolLanguage` or `IsCobolFile` is meaningful for the input file not really at option level. We could see it from a different angle like `EnableTypeCobol` which sounds more like an option.

Reply:
Sur sometime I write boolean (I think java) and completion change it , done in 8b26513